card.css

```diff
@@ -0,0 +1,130 @@
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.7rem;
+}
+.vertical-card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  max-width: 19rem;
+  text-align: center;
+  margin: auto;
+  margin-top: 0.6rem;
+  cursor: pointer;
+}
+.vertical-card img {
```

Comment:
directly manipulating the img tag is often discourage
what u can do is put it inside a div tag and manipulate the div instead.